Makefile

```diff
@@ -363,26 +363,32 @@ localnet-keys:
 localnet-build:
 	@DOCKER_BUILDKIT=1 docker-compose -f tests/localosmosis/docker-compose.yml build
 
-localnet-build-state-export:
-	@docker build -t local:osmosis-se --build-arg ID=$(ID) -f tests/localosmosis/mainnet_state/Dockerfile-stateExport .
-
 localnet-start:
 	@docker-compose -f tests/localosmosis/docker-compose.yml up
 
 localnet-startd:
 	@docker-compose -f tests/localosmosis/docker-compose.yml up -d
 
-localnet-start-state-export:
-	@docker-compose -f tests/localosmosis/mainnet_state/docker-compose-state-export.yml up
-
 localnet-stop:
 	@docker-compose -f tests/localosmosis/docker-compose.yml down
 
 localnet-remove: localnet-stop
 	rm -rf $(PWD)/tests/localosmosis/.osmosisd
 
+localnet-build-state-export:
+	@DOCKER_BUILDKIT=1 docker-compose -f tests/localosmosis/state_export/docker-compose.yml build
+
+localnet-start-state-export:
+	@docker-compose -f tests/localosmosis/state_export/docker-compose.yml up
+
+localnet-startd-state-export:
+	@docker-compose -f tests/localosmosis/state_export/docker-compose.yml up -d
+
+localnet-stop-state-export:
+	@docker-compose -f tests/localosmosis/docker-compose.yml down
+
 localnet-remove-state-export:
-	@docker-compose -f tests/localosmosis/mainnet_state/docker-compose-state-export.yml down
+	rm -rf $(PWD)/tests/localosmosis/state_export/.osmosisd
```

Comment:
Yes, we're using a `.osmosisd` folder within the repo.
Same behavior of regular localOsmosis.

Should we change both?
TBH I prefer keeping it inside the repo but it's not a problem changing.

Reply:
Can we change this on both? From personal experience (today actually), I was confused where it was pulling the data from and had to remake a testnet instead of upgrading it. With the knowledge I have now I realize I could have done the following: 

1. Move the .osmosisd repo to the home folder
2. Checkout v12
3. Move the .osmosisd repo back to the in repo location

This seems like a hassle when we could just keep it using .osmosisd. What are your thoughts though? Totally down to keep it this way if there is a good reason!

Reply:
The osmosis home makes more sense, to be honest.

I like the `.osmosisd` folder within the repo because the initialization step (running `testnetify` and modifying the various tomls) runs only if no  `.osmosisd/config` folder it's found: https://github.com/osmosis-labs/osmosis/blob/main/tests/localosmosis/scripts/setup.sh#L87

It's less likely to have a `.osmosisd` folder already in the GitHub repository, while it's very possible to have already a `.osmosisd` folder.